codegen/core/src/main/java/software/amazon/smithy/python/codegen/ClientGenerator.java

```diff
@@ -553,6 +554,14 @@ await sleep(retry_token.retry_delay)
                                     )
                                 )
                                 logger.debug("Signed HTTP request: %s", context.transport_request)
+
+                                # TODO - Move this to separate resolution/population function
+                                fields = context.transport_request.fields
+                                auth_value = fields["Authorization"].as_string()  # type: ignore
+                                signature = re.split("Signature=", auth_value)[-1]  # type: ignore
+                                context.properties["signature"] = signature.encode('utf-8')
+                                context.properties["identity"] = identity
+                                context.properties["signer_properties"] = auth_option.signer_properties
```

Comment:
Nit: These should probably use the typed properties.

These are generic and I think something reasonable to always have after auth resolution, so I think we can leave them here.

Reply:
I think we're entirely refactoring this with Jordon's upcoming work. Ideally we're not doing this at all going forward.